### PR TITLE
Bump version 7.0.7 to use aiocoap version 0.4.1

### PR DESCRIPTION
**Pull request by janiversen**
This seems to solve some of the stability issues, that have been reported.

### CODE REVIEW

**Comment by janiversen** [comment]
@MartinHjelmare, my first commit here. Can you please help make this available on pipy.org.

Next step is to add typing to the files, so I can remove cast() in the integration.

**Comment by MartinHjelmare** [comment]
Did you see https://github.com/home-assistant-libs/pytradfri/pull/314?

File: pytradfri/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-7.0.6
+7.0.7
```
Comment (MartinHjelmare) [inline pytradfri/VERSION:1]: Our standard is to bump the version separately from other changes, just before releasing.

Comment (janiversen) [reply pytradfri/VERSION:1]: ok, I will close this for now and investigate further on my local system. I did now have the problems reported in #314.

**Pull request closed without merging**